packages/zk-eddsa-event-ticket-pcd/test/ZKEdDSAEventTicketPCD.spec.ts

```diff
@@ -501,6 +533,12 @@ describe("ZKEdDSAEventTicketPCD should work", function () {
     await testVerifyBadClaim(validPCD, (claim: ZKEdDSAEventTicketPCDClaim) => {
       claim.partialTicket.ticketCategory = TicketCategory.PcdWorkingGroup;
     });
+    await testVerifyBadClaim(validPCD, (claim: ZKEdDSAEventTicketPCDClaim) => {
```

Comment:
There are more test cases of this form with one subcase per field which should be expanded with the new fields:
  should not prove with incorrect args
  should not verify a proof with incorrect partialTicket claims
  should not verify a proof with incorrectly revealed partialTicket claims
  should not verify a proof with incorrect non-ticket claims

Reply:
I've done the first three. I don't think there's anything to add for the test relating to non-ticket claims, since the new claims are part of the ticket.

Reply:
You're right, sorry, I misinterpreted that last one as non-revealed rather than non-ticket.